migtests/tests/import-file/run-import-file-test

```diff
@@ -70,6 +70,28 @@ main() {
 	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter '\t' \
 			--file-table-map "accounts_data.csv:accounts" --has-header=true --batch-size 10000 --null-string "\N"
 
+	#Test for Large rows and non-default batch size bytes (i.e. 17MB for row sizes of 15MB,11MB,5KB,12MB,9.9MB)
```

Comment:
What is the order of these rows? The batch numbers in expected_batch_files_large_rows suggest that they do not match this order here, because first batch is 9MB, for example.

Reply:
So, no the first batch is not the one numbered wit h 0 batchNum, that is some logic I believe to confirm if the file is completely splitted or not (or whether batch generation is completed) for that why we have taken but the `0` batchNum is the last batch num
```
...
LAST_SPLIT_NUM = 0
....
if isLastBatch {
    batchNumber = LAST_SPLIT_NUM
}
...
if batch.Number == LAST_SPLIT_NUM {
	fileFullySplit = true
}
...
if batch.Number == LAST_SPLIT_NUM {
	batchGenerationCompleted = true
}
```